nhais-adaptor/adaptor/incoming/operation_definition_adaptor.py

```diff
@@ -0,0 +1,58 @@
+from edifact.incoming.models.interchange import Interchange
+import adaptor.fhir_helpers.fhir_creators as creators
+from datetime import datetime
+
+
+def format_date_time(edifact_date_time):
+    """
+    format the edifact date time to a fhir format
+    :param edifact_date_time: The incoming date time stamp from the interchange header
+    :return: The formatted date time stamp
+    """
+    current_format = "%y%m%d:%H%M"
+    desired_format = "%Y-%m-%d %H:%M"
+    formatted_date = datetime.strptime(edifact_date_time, current_format).strftime(desired_format)
+    return formatted_date
+
+
+def create_operation_definition(interchange: Interchange):
+    """
+    Create a fhir operation definition from the incoming interchange
+    :param interchange: The incoming interchange
+    :return: OperationDefinition:

```

Comment:
Yeah the docstring needs updating. Umm I'm kinda tempted to make these statically typed. So it's obvious what's getting returned and always up to date. I'm not sure if that is the pythonic way but I think it's valuable in an adaptor when we are taking in one type and transforming it to another type. What do you guys think?

Reply:
Could you use [type hints](https://lwn.net/Articles/640359/) for this? It would help users of the code and IDEs understand what's going to be returned, but still be pythonic.